manifests/kustomization.yaml

```diff
@@ -2,14 +2,20 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: bennuhp
 resources:
+  # - ./controllers/sealed-secret-controller.yaml
+  - ./controllers/ingress-nginx-controller.yaml
   - ./namespace.yaml
   - ./sealed-secrets.yaml
   - ./nfs-server.yaml
   - ./staticfiles.yaml
   - ./mysql.yaml
   - ./django.yaml
   - ./nginx.yaml
+  - ./ingress.yaml
 configMapGenerator:
   - name: nginx-etc
     files:
       - ./configs/nginx.conf
+replicas:
```

Comment:
note: for scheduling ingress-nginx-controller pods on each node, we have updated from default manifest.